### PR TITLE
server card ui polish

### PR DESCRIPTION
**Pull request by ohnocassio**
- adjusted max-width on cards to make margins more consistent on pixel
and iphone
- border radius on actions
- top and bottom content padding (20%) for tall/better UI experience on more
devices

### CODE REVIEW

File: www/ui_components/server-card.html [hunk 1]
```diff
@@ -57,6 +57,7 @@
       }
 
       .card-content {
+        padding: 20% 0 20% 0;
```
Comment (albertolalama) [inline www/ui_components/server-card.html]: This makes the card too big for smaller displays - I believe the smallest we target is iPhone 4 (320x480).

screen shot 2018-03-23 at 6 13 24 pm

I played around with some values and suggest we use the following rules:
```css 
     .card-content {
        text-align: center;
        padding: 10% 0;
      }

      @media (min-height: 600px) {
        .card-content {
          padding: 20% 0;
        }
      }
```
What do you think?

**Comment by albertolalama** [comment]
Also, I'm thinking we should probably also scale up the card image in `www/ui_components/server-connection-viz.html` (scaled 1.2x):
```css
    @media(min-height: 600px) {
        #sm, #sm-grey {
          top:  72px;
          left: 72px;
          height: 48px;
          width:  48px;
        }

        #md, #md-grey {
          top:  36px;
          left: 36px;
          height: 120px;
          width:  120px;
        }

        #lg, #lg-grey, #lg-zero {
          top:  0;
          left: 0;
          height: 192px;
          width:  192px;
        }
      }
```
Thoughts?

**Comment by ohnocassio** [comment]
brilliant! I forgot all about iPhone 4, good call. Good suggestion scaling up the server viz, makes everything more balanced.

File: www/ui_components/server-card.html (hunk 1 above)
Comment (ohnocassio) [reply www/ui_components/server-card.html]: added

**Pull request merged**